README.md

```diff
@@ -21,33 +21,36 @@ If you haven't already, [sign up for a free Datadog account](https://www.datadog
 
 > **Make sure that APM is enabled.** The agent does not have APM enabled by default so make sure [to enable it](https://docs.datadoghq.com/tracing/setup/?tab=agent630#agent-configuration).
 
-### Installation
+### Installing the extension
 
-The PHP tracer is composed of a PHP extension and a Composer package. You'll need to install both in order to start tracing your PHP projects.
+You can install the extension from a package download. First [download the appropriate package](https://github.com/DataDog/dd-trace-php/releases) from the releases page. Then install the package with one of the commands below.
 
-#### Composer installation
+```bash
+# using RPM package (RHEL/Centos 6+, Fedora 20+)
+$ rpm -ivh datadog-php-tracer.rpm
 
-First we'll install the Composer package.
+# using DEB package (Debian Jessie+ , Ubuntu 14.04+)
+$ deb -i datadog-php-tracer.deb
 
-```bash
-$ composer require datadog/dd-trace
+# using APK package (Alpine)
+$ apk add datadog-php-tracer.apk --allow-untrusted
+
+# using tar.gz archive (Other distributions using libc6)
+$ tar -xf datadog-php-tracer.tar.gz -C /
+  /opt/datadog-php/bin/post-install.sh
 ```
 
-#### Installing the extension
+## Beta support  for PECL
 
-Next we'll install the `ddtrace` extension. The easiest way to install the extension is from [PECL](https://pecl.php.net/package/datadog_trace).
+Preliminary beta  support for PECL installation is required [PECL](https://pecl.php.net/package/datadog_trace).
```

Comment:
That dang MacBook space bar strikes again! 😱😄
```suggestion
Preliminary beta support for PECL installation is required [PECL](https://pecl.php.net/package/datadog_trace).
```
A total nit-pick, but I couldn't   resist since  . mine is doing the   same dang  thing! 😭Somebody throw me an air can!